keyboards/boston_meetup/2019/matrix.c

```diff
@@ -0,0 +1,160 @@
+#include <string.h>
```

Comment:
Why does this need a custom matrix?

Reply:
i think this is needed for the encoder? not 100% sure, just been using custom matrix since moving to arm

Reply:
Ah, yeah.  Well it was using a custom matrix, but it shouldn't need to do so anymore.  The issues that required a custom matrix should be fixed now, so this shouldn't be necessary.  Also, encoder support is present in the default matrix, too.

Reply:
custom matrix removed, standard matrix def done and tested working on the macropad